Marlin/language_sk_utf8.h

```diff
@@ -0,0 +1,350 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (C) 2016 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (C) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * Slovak
+ * UTF-8 for Graphical Display
+ *
+ * LCD Menu Messages
+ * See also https://github.com/MarlinFirmware/Marlin/wiki/LCD-Language
+ *
+ * Translated by Michal Holeš, Farma MaM
+ * http://www.facebook.com/farmamam
+ *
+ */
+#ifndef LANGUAGE_SK_UTF_H
+#define LANGUAGE_SK_UTF_H
+
+#define MAPPER_C3C4C5_SK
+#define DISPLAY_CHARSET_ISO10646_SK
```

Comment:
This CHARSET should be added to the list in `language.h`, at least as a placeholder if there won't be any custom font needed. However, it appears like this language would benefit from a custom font (for characters like `ť`).